_episodes/01-welcome.md

```diff
@@ -8,8 +8,10 @@ questions:
 - "What will we cover (or not) in this workshop?"
 - "What prior knowledge do workshop participants have?"
 objectives:
+- "Explain what made their favorite class(es) so great." 
```

Comment:
Or "Reflecting on previous learning experiences to identify components of successful teaching" ?